src/include/buffer.h

```diff
@@ -1066,6 +1066,9 @@ struct error_code;
 
     void claim(list& bl);
     void claim_append(list& bl);
```

Comment:
The entire `claim_...` family deserves deprecation, I believe. `claim_append()` is basically pre-C++11 way to say `dst.append(std::move(src))`.

Reply:
Yeah, that's pretty much what I thought and that's the reason I didn't include this trivial change in the msgr2.1 PR -- I assumed you'd want to sweep the codebase and get rid of `claim()` and `claim_append()`.

Reply:
@rzarzynski do you plan to mark `claim_append()` and friends with `[[deprecated]]` specifier in a follow-up PR?